opa_client/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "opa_client"
+version = "0.1.0"
+edition = "2021"
+authors = ["Garry O'Donnell <garry.o'donnell@diamond.ac.uk>"]
+description = "An Open Policy Agent RESTful client"
```

Comment:
This confused me a bit because there is a module called `graphql`, took me a moment to realise it's just a guard, but I think the description should still reflect that it's more than a REST client

Reply:
Yeah, probably worth a mention of the other utilities